skos/ebu_ParentalGuidanceCodeCS_amgext.rdf

```diff
@@ -968,6 +970,7 @@
       rdf:about="http://www.ebu.ch/metadata/ontologies/skos/ebu_ParentalGuidanceCodeCS#_7">
       <rdf:type rdf:resource="http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#TargetAudience"/>
       <skos:prefLabel xml:lang="en">BR-DJCTQ</skos:prefLabel>
+      <skos:altLabel xml:lang="pt-BR">Departamento de JustiÃ§a, ClassificaÃ§Ã£o, TÃ­tulos e QualificaÃ§Ã£o</skos:altLabel>

```

Comment:
@gaurav-amagi some unicode character issue here "Departamento de Justiça, Classificação, Títulos e Qualificação "
https://www.jusbrasil.com.br/artigos/busca?q=DEPARTAMENTO+DE+JUSTI%C3%87A%2C+CLASSIFICA%C3%87%C3%83O%2C+T%C3%8DTULOS+E+QUALIFICA%C3%87%C3%83O

Reply:
Yes @shashi-banger we have asked Gracenote for a better list to avoid this problem